File: src/FoodOrderSystem.App/ClientApp/src/app/order/checkout.model.ts
import {StoredCartDishModel} from '../cart/stored-cart-dish.model';

export class CheckoutModel {
  public givenName: string;
  public lastName: string;
  public street: string;
  public addAddressInfo: string;
  public zipCode: string;
  public city: string;
  public phone: string;
  public email: string;
  public orderType: string;
  public restaurantId: string;
  public cartDishes: StoredCartDishModel[];
  public comments: string;
  public paymentMethodId: string;
  public serviceTime: string;
}

File: src/Gastromio.App/Models/CheckoutModel.cs
using System;
using System.Collections.Generic;

namespace Gastromio.App.Models
{
    public class CheckoutModel
    {
        public string GivenName { get; set; }

        public string LastName { get; set; }

        public string Street { get; set; }

        public string AddAddressInfo { get; set; }

        public string ZipCode { get; set; }

        public string City { get; set; }

        public string Phone { get; set; }

        public string Email { get; set; }

        public string OrderType { get; set; }

        public string RestaurantId { get; set; }

        public List<CartDishInfoModel> CartDishes { get; set; }

        public string Comments { get; set; }

        public Guid PaymentMethodId { get; set; }

        public DateTimeOffset? ServiceTime { get; set; }
    }
}
